pod/perldeprecation.pod

```diff
@@ -14,6 +14,22 @@ features are available.
 The deprecated features will be grouped by the version of Perl in
 which they will be removed.
 
+=head2 Perl 5.40
+
+=head3 Downgrading a C<use VERSION> to below v5.11
+
+Once Perl has seen a C<use VERSION> declaration that requests a version
+C<v5.11> or above, a subsequent second declaration that requests an earlier
+version will print a deprecation warning. For example,
```

Comment:
Once again it'd be nice if this spelled out why we are deprecating this

Reply:
I see it below in perldiag, but if the reason was in all places we spelled it out (or a link of where to find the reason)

Reply:
Much wording added and re-pushed.